package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -726,6 +801,642 @@ def __setstate__(self, state):
+    positions : bool (optional)
+        Write positions into the trajectory [``True``]
+    velocities : bool (optional)
+        Write velocities into the trajectory [``True``]
+    forces : bool (optional)
+        Write forces into the trajectory [``True``]
+    timeunit : str (optional)
+        Option to convert values in the 'time' dataset to a custom unit,
+        must be recognizable by MDAnalysis
+    lengthunit : str (optional)
+        Option to convert values in the 'position/value' dataset to a
+        custom unit, must be recognizable by MDAnalysis
+    velocityunit : str (optional)
+        Option to convert values in the 'velocity/value' dataset to a
+        custom unit, must be recognizable by MDAnalysis
+    forceunit : str (optional)
+        Option to convert values in the 'force/value' dataset to a
+        custom unit, must be recognizable by MDAnalysis
+    author : str (optional)
+        Name of the author of the file
+    author_email : str (optional)
+        Email of the author of the file
+    creator : str (optional)
+        Software that wrote the file [``MDAnalysis``]
+    creator_version : str (optional)
+        Version of software that wrote the file
+        [:attr:`MDAnalysis.__version__`]
+
+    Raises
+    ------
+    RuntimeError
+        when `H5PY`_ is not installed
+    ValueError
+        when ``n_atoms`` is 0
+    ValueError
+        when ``chunks=False`` but the user did not specify ``n_frames``
+    ValueError
+        when ``positions``, ``velocities``, and ``forces`` are all
+        set to ``False``
+    TypeError
+        when the input object is not a :class:`Universe` or
+        :class:`AtomGroup`
+    IOError
+        when ``n_atoms`` of the :class:`Universe` or :class:`AtomGroup`
+        being written does not match ``n_atoms`` passed as an argument
+        to the writer
+    ValueError
+        when any of the optional ``timeunit``, ``lengthunit``,
+        ``velocityunit``, or ``forceunit`` keyword arguments are
+        not recognized by MDAnalysis
+
+    Notes
+    -----
+
+    By default, the writer will write all available data (positions,
+    velocities, and forces) if detected in the input
+    :class:`~MDAnalysis.coordinates.base.Timestep`. In addition, the settings
+    for ``compression`` and ``compression_opts`` will be read from
+    the first available group of positions, velocities, or forces and used as
+    the default value. To write a file without any one of these datsets,
+    set ``positions=False``, ``velocities=False``, or ``forces=False``.
+
+    .. rubric:: Units
+
+    The H5MD format is very flexible with regards to units, as there is no
+    standard defined unit for the format. For this reason, :class:`H5MDWriter`
+    does not enforce any units. The units of the written trajectory can be set
+    explicitly with the keyword arguments `lengthunit`, `velocityunit`, and
+    `forceunit`. If units are not explicitly specified, they are set to the
```

Comment:
```suggestion
    explicitly with the keyword arguments ``lengthunit``, ``velocityunit``, and
    ``forceunit``. If units are not explicitly specified, they are set to the
```

Do we want double ticks here? There's double ticks above for other parameters I _think_?

Reply:
I _think_ that when we refer to parameters in docstrings we use single ticks, i.e., the reST default role. If it's anything else code-like for which there's no reST role in sphinx then we use the "monospace" double-backticks.

(I didn't look at the code context here so I don't know if `lengthunit` and friends are parameters/kwargs of a method or no.)

Reply:
They're parameters of H5MDWriter's `__init__()`, so I _think_ they should get double back-ticks, or not?

Reply:
single back ticks

Reply:
I've been way too liberal with doubles... fixing now